<!DOCTYPE html>
<html lang="">
<head>
    <link rel="stylesheet" type="text/css" href="style.css">
    <meta charset="UTF-8">
    <title>SRP - Alternative Approach</title>
        <meta name="Author" content=""/>
        <link rel="stylesheet" type="text/css" href="style.css">
</head>
<body>

<div class="navbar">
    <a href="index.html">Main</a>
    <a href="srpabout.html">About RNA-seq</a>
    <a href="srporig.html">Original Approach</a>
    <a class="active" href="srpalt.html">Alternative Approach</a>
    <a href="srpsearch.html">Gene Search</a>
    <a href="srpref.html">References</a>
</div>


<div class = "title">
<h2>
    Alternative approach
</h2>
</div>

<div class = "main">
<h3>
        Loupe Browser
</h3>
<p>
As it does not involve the need to do any programming, we used the desktop application Loupe Browser to analyse data from CellRanger for interactive visualization. Loupe Browser takes any .cloupe file generated by CellRanger as an input. Prior to  analysis, we combined all samples by aggregating the outputs from cellranger count using the aggr pipeline (cellranger aggr). Then, we uploaded the generated .cloupe file to Loupe Browser. The cell ranger .cloupe file contains information about, inter alia, gene expression for cells in the sample and gene annotation for the transcriptome reference.
</p>
<h3>
Identifying Cell Types
</h3>
<p>
The 'Identifying cell types' Loupe Browser feature was used to find cell-types within the dataset. To do so, we used a genes_set_input_file dataset containing the name of the gene set followed by the gene IDs, which is provided by authors and available on github. We successfully created sets of markers containing the following cell types: oligodendrocytes (ODC), excitatory, microglia, oligodendrocytes precursor cells (OPC), astrocytes, endothelial, inhibitory, ependymal, CADPS2, GABA and dopaminergic neurons (DaNs).
</p>
<h3>
Reclustering Workflow
</h3>
<p>
Initially, the .cloupe gene expression dataset included 54,353 barcodes derived from a set of all samples. We performed a reclustering workflow in Loupe Browser by applying thresholds used by Smajic et al., 2020. Minimum thresholds for UMIs and features expression were set to 1500 and 1000 respectively. In order to filter cells by mitochondrial fraction, we used the human reference genome (hg19) and we set the maximum threshold to 10%. 43,641 barcodes were included in further analysis after performing reclustering.
</p>
<p>
<b>Figure 1:</b> UMAP embeddings prior to reclustering workflow
</p>
<img src="figures/Fig1.png" width="75%">
<p>
<b>Figure 2:</b> UMAP embeddings after the reclustering workflow
</p>
<img src="figures/Fig2.png" width="75%">
<h3>
Manual cell cluster annotation
</h3>
<p>
By applying information provided by Smajic et al., 2020 in the article, we manually annotated 10 K-means clusters with most abundant cell types. We were looking at expression of gene characteristics for each of the 12 imported cell types: ODC (characterized by the expression of MOBP),  excitatory (SLC17A6), microglia (CD74), pericytes (PDGFRB), OPC (VCAN), astrocytes (AQP4), endothelial (CLDN5), inhibitory (GAD2), ependymal (FOXJ1), CADPS2 (CADPS2), GABAergic (GAD2/GRIK1) and DaNs (TH). We successfully annotated all custers as shown on Figure 3.
</p>
<p>
<b>Figure 3:</b> Annotated clusters
</p>
<img src="figures/Fig3.png" width="75%">
<h3>
Comparison of cell type distribution
</h3>
<p>
To investigate the cell-type composition difference between IPD and control samples, we retrieved the mean distribution of each cell-type per sample from Loupe Browser (Log 2 Avg Count). Then, we calculated the average distribution separately for IPD and control samples. Those data showed an increase in the fraction of microglia and astrocytes (see Figure 4&5) and a decrease in fraction of ODC in IPD samples compared to controls (Figure 6). 
</p>
<p>
<table>
    <tr>
        <th>Cell-type</th>
        <th>IPD (Log2 Avg Count)</th>
        <th>Controls (Log2 Avg Count)</th>
    </tr>
    <tr>
        <td>Microglia</td>
        <td>0.743</td>
        <td>0.591</td>
    </tr>
    <tr>
        <td>Astrocytes</td>
        <td>0.961</td>
        <td>0.829</td>
    </tr>
    <tr>
        <td>Oligodendrocytes</td>
        <td>1.450</td>
        <td>1.548</td>
    </tr>
    <tr>
        <td>GABA</td>
        <td>0.899</td>
        <td>0.919</td>
    </tr>
    <tr>
        <td>DaNs</td>
        <td>0.770</td>
        <td>0.8</td>
    </tr>
    <tr>
        <td>Ependymal</td>
        <td>0.750</td>
        <td>0.7</td>
    </tr>
    <tr>
        <td>Inhibitory</td>
        <td>0.756</td>
        <td>0.790</td>
    </tr>
    <tr>
        <td>Endothelial</td>
        <td>0.520</td>
        <td>0.587</td>
    </tr>
    <tr>
        <td>Excitatory</td>
        <td>0.795</td>
        <td>0.830</td>
    </tr>
    <tr>
        <td>OPC</td>
        <td>1.036</td>
        <td>1.077</td>
    </tr>
    <tr>
        <td>CADPS2</td>
        <td>0.689</td>
        <td>0.733</td>
    </tr>
</table>

</p>
<p>
<b>Figure 4:</b> Violin plot presenting an average distribution of Microglia across all samples
</p>
<img src="figures/Fig4.png" width="75%">
<p>
<b>Figure 5:</b> Violin plot presenting an average distribution of Astrocytes across all samples
</p>
<img src="figures/Fig5.png" width="75%">
<p>
<b>Figure 6:</b> Violin plot presenting an average distribution of ODC across all samples
</p>
<img src="figures/Fig6.png" width="75%">
<p>
In conclusion, our alternative analysis confirmed findings presented by Smajic et al., 2020. We showed increase of microglia and astrocyte abundance and a decreased fraction of ODC in IPD samples compared to controls. 
</p>
</div>
</body>
</html>
