<!DOCTYPE html>
<html lang="">
<head>
    <link rel="stylesheet" type="text/css" href="style.css">
    <meta charset="UTF-8">
    <title>SRP - References</title>
        <meta name="Author" content=""/>
        <link rel="stylesheet" type="text/css" href="style.css">
</head>
<body>


<div class="navbar">
    <a href="index.html">Main</a>
    <a href="srpabout.html">About RNA-seq</a>
    <a href="srporig.html">Original Approach</a>
    <a href="srpalt.html">Alternative Approach</a>
    <a href="srpsearch.html">Gene Search</a>
    <a class="active" href="srpref.html">References</a>
</div>

<div class="title">
<h2>
    References
</h2>
</div>


<div class="main">
<h3>
	Original Paper
</h3>
<p>
S. Smajić, C. A. Prada-Medina, Z. Landoulsi, C. Dietrich, J. Jarazo, J. Henck, S. Balachandran, S. Pachchek, C. M. Morris, P. Antony, B. Timmermann, S. Sauer, J. C. Schwamborn, P. May, A. Grünewald, M. Spielmann. Single-cell sequencing of the human midbrain reveals glial activation and a neuronal state specific to Parkinson’s disease. medRxiv 2020.09.28.20202812 DOI: https://doi.org/10.1101/2020.09.28.20202812
<h3>
        Software
</h3>
<p>
Cell Ranger: <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger</a>
</p>
<p>
Loupe Browser: <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/visualization/latest/what-is-loupe-cell-browser">https://support.10xgenomics.com/single-cell-gene-expression/software/visualization/latest/what-is-loupe-cell-browser</a>
</p>
<p>
R: <a href="https://www.r-project.org/">https://www.r-project.org/</a>
</p>
<p>
Seurat: <a href="https://satijalab.org/seurat/articles/get_started.html">https://satijalab.org/seurat/articles/get_started.html</a>
</p>
<h3>
	Other references
</h3>
<p>
Bio, D., 2021. Single Nuclei RNA-Seq (sNuc-Seq) | Dolomite Bio Technology. [online] Dolomite Bio. Available at: <https://www.dolomite-bio.com/applications/single-nuclei-rna-seq/>.
</p>
<p>
Denisenko, E., Guo, B.B., Jones, M., Hou, R., de Kock, L., Lassmann, T., Poppe, D., Clément, O., Simmons, R.K., Lister, R., Forrest, A.R.R., 2020. Systematic assessment of tissue dissociation and storage biases in single-cell and single-nucleus RNA-seq workflows. Genome Biology. 21, 130.
</p> 	
<p>
Kukurba, K. and Montgomery, S., 2015. RNA Sequencing and Analysis. Cold Spring Harbor Protocols, 2015(11), p.pdb.top084970. 	
</p>
<p>
Lake, B.B., Codeluppi, S., Yung, Y.C., Gao, D., Chun, J., Kharchenko, P.V., Linnarsson, S., Zhang, K., 2017. A comparative strategy for single-nucleus and single-cell transcriptomes confirms accuracy in predicted cell-type expression from nuclear RNA. Scientific Reports. 7, 6031. 
</p>
<p>
Tasic, B., 2018. Single cell transcriptomics in neuroscience: cell classification and beyond. Current Opinion in Neurobiology, 50, pp.242-249.
</p>
<p>
Wang, Z., Gerstein, M. and Snyder, M., 2009. RNA-Seq: a revolutionary tool for transcriptomics. Nature Reviews Genetics, 10(1), pp.57-63.
</p>
</div>

</body>

</html>

